nemo/collections/asr/parts/utils/vad_utils.py

```diff
@@ -955,33 +967,50 @@ def pred_rttm_map(vad_pred: str, groundtruth_RTTM: str, vad_pred_method: str = "
 
 def plot(
     path2audio_file: str,
-    path2_vad_pred: str,
-    path2ground_truth_label: str = None,
+    path2_vad_pred: Optional[str] = None,
+    path2groundtruth_rttm: Optional[str] = None,
+    groundtruth_labels: Optional[str] = None,
+    sample_rate: int = 16000,
     offset: float = 0,
     duration: float = None,
     threshold: float = None,
     per_args: dict = None,
+    unit_frame_len: float = 0.01,
+    label_repeat: int = 1,
+    xticks_step: int = 5,
 ) -> ipd.Audio:
     """
-    Plot VAD outputs for demonstration in tutorial
+    Plot Audio and/or VAD output and/or groundtruth labels for visualization
     Args:
         path2audio_file (str):  path to audio file.
         path2_vad_pred (str): path to vad prediction file,
-        path2ground_truth_label(str): path to groundtruth label file.
+        path2groundtruth_rttm(str): path to groundtruth RTTM file.
+        ground_truth_labels(str): a list of groundtruth label.
+        sample_rate (int): sample rate of audio file.
+        offset (float): offset in seconds.
+        duration (float): duration in seconds.
         threshold (float): threshold for prediction score (from 0 to 1).
         per_args(dict): a dict that stores the thresholds for postprocessing.
+        unit_frame_len (float): unit frame length in seconds for VAD predictions.
+        label_repeat (int): repeat the label for this number of times to match different frame lengths in preds and labels.
+        xticks_step (int): step size for xticks.
     """
     plt.figure(figsize=[20, 2])
-    UNIT_FRAME_LEN = 0.01
 
-    audio, sample_rate = librosa.load(path=path2audio_file, sr=16000, mono=True, offset=offset, duration=duration)
+    audio, sample_rate = librosa.load(
```

Comment:
[Comment ID: a03 - Please change these lines to use `AudioSegment.from_file()`]
Please replace 1000~1004 with `AudioSegment.from_file()`.

Reply:
we don't need AudioSegment for visualization, librosa is fine